### PR TITLE
SPSH-941

### PR DESCRIPTION
**Pull request by AlexanderUngefug**

### CODE REVIEW

**Review by marode-cap — APPROVED** [review]

File: src/themes/schulportal/login/template.ftl [hunk 1]
```diff
@@ -207,19 +207,19 @@
             </a>
             <a
                 class="footer-item"
-                href="https://spsh.staging.spsh.dbildungsplattform.de/impressum_datenschutzerklaerung.html"
+                href="${client.baseUrl}/impressum_datenschutzerklaerung.html"
```
Comment (YoussefBouch) [inline src/themes/schulportal/login/template.ftl:210]: This won't work if the baseUrl in the realm.json isn't `"baseUrl": "${KC_ROOT_URL}"`, so either add that or revert this change in this file to avoid merge problems. 

SPSH-1210 will take care of making those footers dynamic

**Review by YoussefBouch — CHANGES_REQUESTED** [review]

**Review by YoussefBouch — APPROVED** [review]

**Review by YoussefBouch — APPROVED** [review]
